classes/UpgradeSelfCheck.php

```diff
@@ -32,6 +32,21 @@
 
 class UpgradeSelfCheck
 {
+    /**
+     * PHP and PrestaShop Compatibles Versions
+     */
+    const PHP_PS_VERSIONS = ['1.6.x' => ['5.2', '5.3', '5.4', '5.5', '5.6', '7.0', '7.1'],
```

Comment:
Hello, if we use [version_compare](https://www.php.net/manual/en/function.version-compare.php) we'll be able to store only the lower and upper limit, not each version.

And if tomorrow they decide to publish a PHP 7.5 or a PHP 5.8 😆  it will work.

Reply:
Yes i do this, but i say in the PR description : 

`Acutaly i use a constant, @marionf say to use a chanel from PrestaShop Api but he don't exist.`

Reply:
We can use version_compare with a range. It will become
```
const PHP_PS_VERSIONS = ['1.6.x' => ['5.2.0', '7.1.0']], ...
```

Reply:
Before  using version_compare.

In my last screenshot there are all compatible php versions.

So I would replace with 5.2 to 7.1 ?

Reply:
> Acutaly i use a constant, @marionf say to use a chanel from PrestaShop Api but he don't exist.

Yes, we talked about that with @eternoendless, we should create it (related to this [issue](https://github.com/PrestaShop/PrestaShop/issues/25951))